### ISSUE TITLE
[1.6.0] repeatability discrepancies between fields.json and XSDs

### ISSUE THREAD

**Issue by DeMol-EE** [OP-TED/eForms-SDK#417]
While working on an engine to turn physical models into conceptual models (for the purpose of copying information from one notice to another), we ran into problems for elements that repeated in XML but that are not repeatable according to fields.json.

For example, processing "cn_24_maximal.xml" (SDK version 1.6.0) reports that the following (conceptual) nodes have multiple instances of a (conceptual) field that is not repeatable:

```
ND-SecondStageCriterionParameter - BT-7532-Lot
ND-SecondStageCriterionParameter - BT-752-Lot
ND-LotRestrictedDocuments - BT-615-Lot
ND-LotDocumentsReference - BT-15-Lot
```

Indeed, in cn_24_maximal.xml, we see that `<efac:SelectionCriteria>` contains `<efac:CriterionParameter>` twice (for all 3 lots), which is allowed according to XSD (efac’s definition of `CriterionType` lists `efac:CriterionParameter` as maxOccurs unbounded).
In fields.json, however, we see that `ND-SecondStageCriterionParameter` (which has `efac:CriterionParameter` as relative XPath) is considered non-repeatable.

This leads to issues because our conceptual model only creates one node for "ND-SecondStageCriterionParameter" (as it is not repeatable) and thus both values for BT-752-Lot are added as child to that one node, which we can no longer turn into valid XML.

Could this be a possible mistake in the SDK or are we doing something wrong?

**Comment by YvesJo** [comment]
Hi,
The node ND-SecondStageCriterionParameter should be repeatable and this will get fixed.
The repeatable aspect of a field is expressed in the context of its parent node. There may exit more than one reference to procurement documents, hence ND-LotRestrictedDocuments and ND-LotDocumentReference are repeatable. However these nodes may only contain one URL (BT-15 or BT-615).
KR.

**Comment by pankajgulia111** [comment]
Hi , Similar issue with fields  "BT-743-Lot", "BT-764-Lot", "BT-736-Lot" , with there relative xpath  
`"cac:ContractExecutionRequirement/cbc:ExecutionRequirementCode[@listName='einvoicing']"`,
`"cac:ContractExecutionRequirement/cbc:ExecutionRequirementCode[@listName='ecatalog-submission']"`,
`"cac:ContractExecutionRequirement/cbc:ExecutionRequirementCode[@listName='reserved-execution']"`. 
Above fields have `repeatable.value` equals false in fields.json .
In cn_24_minimal.xml we can see that `<cac:TenderingTerms>` contains  `<cac:ContractExecutionRequirement>` thrice. However you can see these fields have `cac:ContractExecutionRequirement` in there relative xpath .
This result in creating one node of `cac:ContractExecutionRequirement` and all three fields as its child and thus an invalid xml file.
Do we treat these repeatable ?

**Comment by YvesJo** [comment]
Fixed with SDK 1.9
